sdk/openai/Azure.AI.OpenAI/CHANGELOG.md

```diff
@@ -4,11 +4,17 @@
 
 ### Features Added
 
+- The SDK now directly supports alternative authentication audiences, including Azure Government. This can be specified by providing an appropriate `AzureOpenAIAudience` value to the `AzureOpenAIClientOptions.Audience` property when creating a client. See the client configuration section of the readme for more details.
```

Comment:
```suggestion
- The library now directly supports alternative authentication audiences, including Azure Government. This can be specified by providing an appropriate `AzureOpenAIAudience` value to the `AzureOpenAIClientOptions.Audience` property when creating a client. See the client configuration section of the README for more details.
```